### DISCUSSION TITLE
Timer get reset all the time

### DISCUSSION THREAD

**Discussion by menisadi** [tabwrangler/tabwrangler#482] [General]
I installed the extension (Firefox) and set the timer to 1440 (a day). I noticed that after a week, no tab got auto-closed. I looked into the tabs-lock section and saw that the timer just keeps getting reset all the time even when I don't open or interact with them, so they are kept alive.

Using latest Firefox version (with Sideberry add-on and multi account containers and some more plugins).

**Comment by Riversiderepeat** [comment]
I hope I'm not stepping in where I have no business but my first instinct about this issue is that you may want to set your timer to 1439 minutes. It is possible that when set to 1440, if the code is using a 24 hour time table (1440 minutes) that because after 1440 minutes it is the same time as it was when you set it, it's not detecting a change in time. Subtracting one minute from your timer in these cases usually fixes the issue.

**Reply by menisadi** [reply]
Interesting. I'll try :)

**Reply by menisadi** [reply]
So I tried, and it still seems that the timers are just reset all the time (literally after 5 seconds). 
I see this when I open the plugin and look at 'tab-lock' section, the timers are going down for 5 seconds and then reset. 
Tried disabling almost all of my plugins with no effect (I didn't disable Sideberry and Firefox Containers as those two are the two which I just can't live without)

**Comment by ssorallen** [comment]
Hi, maintainer here - if the timers are getting reset that means you have more than your minimum number of tabs open. If you set the minimum number of tabs to 5, for example, and have 6 tabs open then the timers will not count down.

**Reply by menisadi** [reply]
Wait, isn't it the opposite?
I see an option: Auto-close inactive tabs if there are more than
Which should like it I set it to 5 the timers should count down whenever I gave more than 5 tabs.
Anyway it was set to 20 which might be the problem. Not sure. I'm trying to set it to 0 and see if it changes anything.

**Reply by menisadi** [reply]
Ok, it looks like it did solve it (timers finally went below 990, starting 1000). 
Thanks!

**Comment by christophberger** [comment]
For the records, I had the same behavior (timer reset after 5 seconds for all tabs), regardless of the countdown time and "close if more than X tabs" setting.

Only when I unchecked "Don't close tabs that are playing audio", the timers continued counting down.

Could be a browser issue, though, as I observe this on Orion, a rather new broswer by Kagi. I'll leave this comment here in case others run over the same issue. 

Browser: Orion Version 0.99.133-beta (WebKit 621.1.2.111.4) on macOS

**Reply by ssorallen** [reply]
v7.9.0 includes a fix for Orion: https://github.com/tabwrangler/tabwrangler/issues/519

**Comment by ssorallen** [comment]
[v7.9.0](https://github.com/tabwrangler/tabwrangler/releases/tag/v7.9.0) includes a fix for Chromium-based browsers like Orion. If you continue to have issues after upgrading please open a new issue.
